setup.cfg

```diff
@@ -6,15 +6,15 @@ description = Analysis of digital elevation models (DEMs)
 keywords = dem, elevation, geoutils, xarray
```

Comment:
In demcompare, the setup files have a header; I'm checking to see if it's necessary here or not.

Reply:
not needed, doesn't need to be overcopyrighted ;) setup follows usually a standard way and no invention to be copyrighted.